saleor/graphql/order/mutations/fulfillments.py

```diff
@@ -97,7 +97,20 @@ class Meta:
     @classmethod
     def clean_lines(cls, order_lines, quantities):
         for order_line, line_quantities in zip(order_lines, quantities):
-            line_quantity_unfulfilled = order_line.quantity_unfulfilled
+
```

Comment:
`order_line.quantity_unfulfilled` is a property on `OrderLine` model. My question is - How should we treat fulfillments which are awaiting for approval?  Should we include them in `order_line.quantity_fulfilled`? 
Currently we have `(order_line.unfulfilled + order_line.fulfilled) == order_line.quantity`.  By changing this in the proposed way, the unfulfilled value doesn't mean - "quantity which doesn't belong to any fulfillment" but "quantity which doesnt belong to any fulfillment OR belong to fulfillment with status awaiting for approval". In proposed way we have to double-check all logic which uses these values - especially features like return, refunds. but I would make sure that this is the best way of handling this. Honestly I am a fan of the order_line's values which determines the status of quantity ( X is unfulfilled, Y is fulfilled).  Maybe we should consider adding a third value to order_line for describing awaiting approveal status.